lib/serve/functions.js

```diff
@@ -104,6 +105,7 @@ function _startEmulator(options) {
   }).then(function(triggers) {
     var promises = _.map(triggers, function(trigger) {
       if (trigger.httpsTrigger) {
+        track('Functions Emulation (Trigger Type)', 'HTTPS');
```

Comment:
Would you separately like to add a top-level event that just tracks that the emulator was run at all? Since this will be per-function and won't provide that info.